src/t8_vtk/t8_vtk_writer.hxx

```diff
@@ -430,6 +423,16 @@ class vtk_writer {
     int *cellTypes = T8_ALLOC (int, num_cells);
     T8_ASSERT (cellTypes != NULL);
 
+    /* Allocate VTK Memory for the arrays */
+    int iMaxCellSize = 20;
```

Comment:
```suggestion
    const int iMaxCellSize = 20;
```
where does the 20 come from?

Reply:
I just did it. Do you have a correct number when thinking about non linear cell types? 
But it should not be an issue. Its just big enough to avoid dynamic reallocation.